docs/src/network.md

```diff
@@ -179,16 +179,12 @@ await using var browser = await BrowserType.LaunchAsync(new()
 });
 ```
 
-When specifying proxy for each context individually, **Chromium on Windows** needs a hint that proxy will be set. This is done via passing a non-empty proxy server to the browser itself. Here is an example of a context-specific proxy:
+Its also possible to specify it per context:
 
 ```js tab=js-test title="playwright.config.ts"
```

Comment:
This snippet should call `browser.newContext()` inside a test, because specifying it in the options makes it global for all contexts.